src/SignalR/server/Core/src/Internal/DefaultHubDispatcher.Log.cs

```diff
@@ -67,14 +66,17 @@ private static class Log
                 LoggerMessage.Define<string>(LogLevel.Trace, new EventId(18, "CompletingStream"), "Stream '{StreamId}' has been completed by client.");
 
             private static readonly Action<ILogger, string, string, Exception> _closingStreamWithBindingError =
-                LoggerMessage.Define<string, string>(LogLevel.Warning, new EventId(19, "ClosingStreamWithBindingError"), "Stream '{StreamId}' closed with error '{Error}'.");
+                LoggerMessage.Define<string, string>(LogLevel.Debug, new EventId(19, "ClosingStreamWithBindingError"), "Stream '{StreamId}' closed with error '{Error}'.");
 
             private static readonly Action<ILogger, Exception> _unexpectedStreamCompletion =
                 LoggerMessage.Define(LogLevel.Debug, new EventId(20, "UnexpectedStreamCompletion"), "StreamCompletionMessage received unexpectedly.");
 
             private static readonly Action<ILogger, Exception> _unexpectedStreamItem =
                 LoggerMessage.Define(LogLevel.Debug, new EventId(21, "UnexpectedStreamItem"), "StreamItemMessage received unexpectedly.");
 
+            private static readonly Action<ILogger, string, Exception> _failedWhileRunningHubMethod =
+                LoggerMessage.Define<string>(LogLevel.Error, new EventId(22, "FailedWhileRunningHubMethod"), "Failed while running hub method '{HubMethod}'.");
```

Comment:
I think it might be better to keep the "FailedInvokingHubMethod" for application failures because that's the one people would be most likely to look for by name. We can create a new debug-level log for "InvalidHubParameters" for client issues.